src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -102,7 +102,7 @@ BusFormat = Format
 # generic/AttributeSetTableModel.java
 #
 attributeChangeInvalidError = Attribut wurde nicht verändert da die Anfrage ungültig war
-FPGA_Supported = FPGA-unterstützt:
+FPGA_Supported = FPGA-unterstützt
```

Comment:
With the above suggested change, this would become "FPGA-Unterstützung"

Reply:
Done